ui/user/src/lib/components/ModelProviders.svelte

```diff
@@ -282,189 +170,59 @@
 		<div class="text-muted">No model providers available</div>
 	{:else}
 		<!-- Default Model Provider Section -->
-		<div class="mb-6">
-			<h4 class="text-md mb-2 font-medium">Default Model</h4>
-			<p class="text-muted mb-3 text-sm">
-				Select the default model for this project. This will be used when no specific model is
-				specified.
-			</p>
-
-			{#if !Object.keys(selectedModels).length}
-				<div class="mb-2 text-sm text-yellow-500">
-					Configure at least one provider and select models to set a default model.
-				</div>
-			{:else}
-				<div>
-					<select
-						class="w-full rounded-md border p-2 text-sm"
-						value={defaultModelProvider && defaultModel
-							? `${defaultModel}|||${defaultModelProvider}`
-							: ''}
-						onchange={(e) => updateDefaultModel(e.currentTarget.value)}
+		<div class={twMerge('mb-6 flex flex-col gap-2', !hasOneModelSelected && 'opacity-50')}>
+			<h4 class="text-md font-medium">Default Model</h4>
+
+			<!-- Not the best approach -->
+			<!-- TODO: Use a 3rd-party UI library or create an internal custom component in the future -->
+			<div class="relative">
+				<select
+					class="border-surface2 dark:bg-surface1 w-full appearance-none rounded-md border px-4 py-4 text-sm"
+					value={defaultModelProvider && defaultModel
+						? `${defaultModel}|||${defaultModelProvider}`
+						: ''}
+					onchange={(e) => updateDefaultModel(e.currentTarget.value)}
+					onclick={() => (isDefaultModelSelectOpen = !isDefaultModelSelectOpen)}
+				>
+					<option class="dark:bg-surface3" value="" disabled
+						>Select the default model for {project.name} project</option
```

Comment:
we do not use the term project. can you change this text to Select the default model for this agent